video.py

```diff
@@ -0,0 +1,70 @@
+import requests
+
+
+class Video:
+    def __init__(self, url="https://retro-video-store-api.herokuapp.com", selected_video=None):
+        self.url = url
+        self.selected_video = selected_video
+
+    # add a video
+    def add_video(self, title="Default Title", release_date=None, total_inventory=0, available_inventory=0):
+        query_params = {
+            "title": title,
+            "release_date": release_date,
+            "total_inventory": total_inventory,
+            "available_inventory": available_inventory
```

Comment:
This looks great, however, I don't think the deployed version of retro store requires an available_inventory parameter for adding a new video so a particular request generated a 404.